component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -201,6 +206,35 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
             if (context.isRetrying()) {
                 retryParam = "&authFailure=true&authFailureMsg=login.fail.message";
             }
+            IdentityErrorMsgContext errorContext = IdentityUtil.getIdentityErrorMsg();
+            IdentityUtil.clearIdentityErrorMsg();
+
+            String errorParam = "";
+            if (Boolean.parseBoolean(showAuthFailureReasonOnLoginPage)) {
+                if (errorContext != null && errorContext.getErrorCode() != null) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("Identity error message context is not null");
+                    }
```

Comment:
No need to check log.isDebugEnabled() since the log doesn't have a string concatenation. isDebugEnabled check is already happens inside log.debug method if there are only 2 params passed
```suggestion
                    log.debug("Identity error message context is not null");
```